external/llvm-project/mlir/include/mlir/Dialect/GPU/GPUOps.td

```diff
@@ -575,34 +575,6 @@ def GPU_PrintfOp : GPU_Op<"printf", [MemoryEffects<[MemWrite]>]>,
   }];
 }
 
-def GPU_SetDefaultDeviceOp : GPU_Op<"set_default_device",
-                                    [MemoryEffects<[MemWrite]>]>,
-    Arguments<(ins I32Attr:$devIndex)> {
-  let summary = "Set default GPU for operations after this by index";
-  let description = [{
-    Operation that sets the current default GPU, using a zero-based index
-    into the set of GPUs on the system. The default GPU setting may be
-    thread-local.
-  }];
-  let assemblyFormat = "attr-dict";
-}
-
-def GPU_PrintfOp : GPU_Op<"printf", [MemoryEffects<[MemWrite]>]>,
```

Comment:
Did the upstream get rid of this op?

Reply:
There were two copies because I changed the format upstream

And SetDefaultDevice never went upstream - that's a TODO item.

Reply:
Sounds good.